tests/sqllogictests/suites/mode/cluster/distributed_copy_into_table.test

```diff
@@ -183,5 +183,24 @@ select count(*) from test_order;
 ----
 4000000
 
+statement ok
```

Comment:
'Explain' is better to know the memory engine is distributed or not, can we add one?

Reply:
explain will always have `distributed insert plan` even in standalone mode, it makes no difference now.